backend/src/handlers/helpful_links_handler.go

```diff
@@ -0,0 +1,134 @@
+package handlers
+
+import (
+	"UnlockEdv2/src/models"
+	"encoding/json"
+	"net/http"
+	"strconv"
+)
+
+func (srv *Server) registerLeftMenuRoutes() []routeDef {
+	axx := models.Feature()
+	return []routeDef{
+		{"GET /api/helpful-links", srv.handleGetHelpfulLinks, false, axx},
+		{"PUT /api/helpful-links", srv.handleAddHelpfulLink, true, axx},
+		{"PATCH /api/helpful-links/{id}/edit", srv.handleEditLink, true, axx},
+		{"PUT /api/helpful-links/toggle/{id}", srv.handleToggleVisibilityStatus, true, axx},
+		{"DELETE /api/helpful-links/{id}", srv.handleDeleteLink, true, axx},
+		{"PUT /api/helpful-links/activity/{id}", srv.handleAddUserActivity, false, axx},
+		{"PUT /api/helpful-links/sort", srv.changeSortOrder, true, axx},
+	}
+}
+
+var HelpfulSortOrder = make(map[uint]string)
+
+func (srv *Server) changeSortOrder(w http.ResponseWriter, r *http.Request, log sLog) error {
+	type reqBody struct {
+		SortOrder string `json:"sort_order"`
+	}
+	var req reqBody
+	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
+		return newJSONReqBodyServiceError(err)
+	}
+	defer r.Body.Close()
+	facilityID := srv.getFacilityID(r)
+	HelpfulSortOrder[facilityID] = req.SortOrder
+	return writeJsonResponse(w, http.StatusOK, "Sort order changed successfully")
+}
+
+func (srv *Server) handleGetHelpfulLinks(w http.ResponseWriter, r *http.Request, log sLog) error {
+	search := r.URL.Query().Get("search")
+	page, perPage := srv.getPaginationInfo(r)
```

Comment:
so if there is an `order_by` query parameter here, and the user is an admin, then we can just persist it. 

By default, the admins request will not have an `order_by` parameter, and we will still always return the current sort order like you are doing already, but we can probably just store it here instead of having it's own endpoint.